### ISSUE TITLE
HTML: minify breaks CSP meta tags by removing necessary space from content attribute

### ISSUE THREAD

**Issue by denisbrodbeck** [tdewolff/minify#170]
Minify breaks valid [Content-Security-Policy](https://developer.mozilla.org/en-US/docs/Web/HTTP/CSP) settings by removing must-have space.

### Input

`<meta http-equiv="Content-Security-Policy" content="default-src 'self'; img-src https://*; child-src 'none';">`

### Output

`<meta http-equiv=content-security-policy content="default-src'self';img-srchttps://*;child-src'none';">`

### Want

`<meta http-equiv=content-security-policy content="default-src 'self';img-src https://*;child-src 'none';">`

Chrome (latest stable) and Firefox (latest stable) log errors to console:
```
Unrecognized Content-Security-Policy directive 'default-src'self''.
Unrecognized Content-Security-Policy directive 'img-srchttps://*'.
Unrecognized Content-Security-Policy directive 'child-src'none''.
```

**minify-version**: 2.3.4

### Example

```golang
package main

import (
	"testing"

	"github.com/tdewolff/minify"
	"github.com/tdewolff/minify/html"
)

const input = `<!DOCTYPE html>
<html lang="en">
<head>
  <meta http-equiv="Content-Security-Policy" content="default-src 'self'; img-src https://*; child-src 'none';">
</head>
<body>
  <main><h1>Hi</h1></main>
</body>
</html>
`

const want = `<!doctype html><html lang=en>
<head>
<meta http-equiv=content-security-policy content="default-src 'self';img-src https://*;child-src 'none';">
</head>
<body>
<main><h1>Hi</h1></main>
</body>
</html>`

func Test_minify(t *testing.T) {
	m := minify.New()
	m.Add("text/html", &html.Minifier{
		KeepDocumentTags:        true,
		KeepWhitespace:          true,
		KeepDefaultAttrVals:     true,
		KeepConditionalComments: true,
		KeepEndTags:             true,
	})

	got, err := m.String("text/html", input)
	if err != nil {
		t.Error(err)
	}
	if got != want {
		t.Errorf("invalid output:\ngot:\n%s\nwant:\n%s\n", got, want)
	}
}
```

**Comment by tdewolff** [comment]
I'll get back to this in two weeks when I have a better internet connection!

**Comment by tdewolff** [comment]
Fixed in 65731477054c4118af51e580c4681137aa2d8534
